packages/typespec-test/test/loadTest/spec/routes.tsp

```diff
@@ -298,7 +298,7 @@ numeric, underscore or hyphen characters.
 
   @summary("Create and start a new test run with the given name.")
   @doc("Create and start a new test run with the given name.")
-  @pollingOperation(LoadTestRun.GetTestRun)
+  // @pollingOperation(LoadTestRun.GetTestRun)
```

Comment:
Prefer to disable this customized pollingOperation because it is not aligned with latest lro template.

https://github.com/Azure/typespec-azure/issues/3619